src/settings.plugin.js

```diff
@@ -371,3 +403,46 @@ function createToggle(params) {
 	return toggleWrapper;
 }
 
+/**
+ * Available directions:
+ * - left
+ * - right
+ *
+ * @param {String} direction
+ * @return {HTMLElement}
+ */
+function createArrowIcon(direction) {
+	var arrowIcon = createSVG(wikiaJWPlayerIcons.back);
+
+	if (direction === 'left') {
+		arrowIcon.classList.add('wikia-jw-settings__back-icon');
+	} else {
+		arrowIcon.classList.add('wikia-jw-settings__right-arrow-icon');
+	}
+
+	return arrowIcon;
+}
+
+function clearListElement(element) {
+	if (element) {
+		while (element.childElementCount > 1) {
+			element.removeChild(element.firstChild);
+		}
+	}
+}
+
+function createSVG(svgHtml) {
```

Comment:
I'd keep these functions in `wikiaJWPlayerSettingsPlugin` scope

Reply:
why? imo it's scope-agnostic, since it doesn't depend on any `wikiaJWPlayerSettingsPlugin` related thing.

Reply:
Because currently they are global, I'd keep only one global function per file or extract all these functions to separate file, for example `helpers.js`

Reply:
k, i'll extract them.